### PR TITLE
feat(tree): add `relative` for `tree.emphasis.focus`

### PR DESCRIPTION
**Pull request by susiwen8**
- https://github.com/apache/echarts/pull/17009

### CODE REVIEW

**Comment by susiwen8** [comment]
@pissang @plainheart Any thoughts?

**Comment by plainheart** [comment]
Since recently we have been updating the website frequently, I think we can wait until v5.3.3 is about to be released so that the future content won't be shown to the developer.

**Comment by susiwen8** [comment]
Ok

File: en/option/partial/emphasis-common.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ When the data is highlighted, whether to fade out of other data to focus the hig
 + `'ancestor'` Focus on all ancestor nodes.
 + `'descendant'` Focus on all descendants nodes.
 {{ /if }}
+{{ if: ${hasRelative} }}
++ `'relative'` since `v${version}`, focus on all ancestor and descendants nodes.
```
Comment (plainheart) [inline en/option/partial/emphasis-common.md:49]: Could we move the version hint to the end of the sentence and wrap it in parentheses? The same to Chinese documentation.

File: zh/option/series/tree.md [hunk 2]
```diff
@@ -204,6 +204,8 @@ center: [115.97, 29.71]
 
 {{ use: partial-focus-blur-scope(
     isTree = true
+    hasRelative = true
+    version = "5.3.3"
```
Comment (plainheart) [inline zh/option/series/tree.md:208]: It seems the `version` variable is not necessary?

Comment (susiwen8) [reply zh/option/series/tree.md:208]: You mean only `hasRelative` here?
The reason I added `version`, it's other `series` who might have same feature in the future version.

**Review by plainheart — APPROVED** [review]

**Pull request merged**